tests/func/utils/test_strict_yaml.py

```diff
@@ -20,7 +20,7 @@
   3 │   cmd: python train.py
 
 Found duplicate key "cmd" with value "python train.py" (original value:\
- "python \ntrain.py"), in line 4, column 5
+ "python\ntrain.py"), in line 4, column 5
```

Comment:
Similarly, if `python` moves above, this could be written as:
```
```suggestion
train.py"), in line 4, column 5
```